README.md

```diff
@@ -95,6 +94,7 @@ The resulting `composer.json` file should look something like this:
         "neos/demo": "@dev",
 
         "neos/contentrepository-development-collection": "@dev",
+        "neos/content-repository-dimensionspace": "dev-master"
```

Comment:
Invalides JSON

Reply:
..aber stimmt, die Abhängigkeit fehlt jetzt im Beispiel?!

Reply:
It's not needed, because `neos/contentrepository-development-collection` and `neos/neos-development-collection@dev-event-sourced-patch` already require it